src/coord/src/lib.rs

```diff
@@ -27,7 +27,7 @@ mod command;
 mod coord;
 mod sink_connector;
 mod timestamp;
-mod util;
+pub mod util;
```

Comment:
ooh, this module shouldn't be public; the `ClientTransmitter` is meant to be private to the `coord` crate; see next comment for a suggestion